src/components/manage/Widgets/DatetimeWidget.jsx

```diff
@@ -140,13 +140,13 @@ class DatetimeWidget extends Component {
       (prevState) => ({
         datetime: prevState.datetime
           ? prevState.datetime.set({
-              hours: time.hours(),
-              minutes: time.minutes(),
+              hours: time?.hours(),
+              minutes: time?.minutes(),
```

Comment:
Wouldn't it be better with a default to 0?

```suggestion
              hours: time?.hours() ?? 0,
              minutes: time?.minutes() ?? 0,
```